desktop/libs/notebook/src/notebook/connectors/trino_tests.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python
+# Licensed to Cloudera, Inc. under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  Cloudera, Inc. licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest import TestCase, mock
+from unittest.mock import MagicMock, patch, Mock
+from useradmin.models import User
+
+from desktop.auth.backend import rewrite_user
+from desktop.lib.django_test_util import make_logged_in_client
+from notebook.connectors.trino import TrinoApi
+
+
+class TestTrinoApi(unittest.TestCase):
+
+  def setUp(self):
+    # Mock user and interpreter
+    self.client = make_logged_in_client(username="hue_test", groupname="default", recreate=True, is_superuser=False)
+    self.user = User.objects.get(username="hue_test")
+    self.interpreter = {
+      'options': {
+        'url': 'http://example.com:8080'
+      }
+    }
+    # Initialize TrinoApi with mock user and interpreter
+    self.trino_api = TrinoApi(self.user, interpreter=self.interpreter)
+
+
+  def test_parse_api_url(self):
+    # Test parse_api_url method
+    api_url = 'http://example.com:8080'
+    expected_result = ('example.com', 8080, 'http')
+    result = self.trino_api.parse_api_url(api_url)
+
+    self.assertEqual(result, expected_result)
```

Comment:
Use assert_equals, assert_true, assert_false etc from nose.tools everywhere?

Reply:
I think both are same but for uniformity we can use imports from nose.tool